src/Core/Components/DataGrid/Columns/ColumnBase.razor

```diff
@@ -26,7 +26,7 @@
             
             if (Sortable.HasValue ? Sortable.Value : IsSortableByDefault())
             {
-                <FluentButton Appearance="Appearance.Stealth" class="col-sort-button" @onclick="@(() => Grid.SortByColumnAsync(this))" aria-label="Sort on this column">
+                <FluentButton Appearance="Appearance.Stealth" class="col-sort-button" @onclick="@(() => Grid.SortByColumnAsync(this))" aria-label="@($"Sort on '{@tooltip}'")" title="@tooltip">
```

Comment:
To be compliant with Accessibility and to avoid localization issue, I would put `aria-label="@tooltip"` This way, the responsibility for writing a correct label is in the hands of the developer.

Reply:
Agreed!. Changing it.